WhoIsThat/WhoIsThat/ViewModels/LoginViewModel.cs

```diff
@@ -72,13 +67,23 @@ public async void TakePhoto()
                 await CrossPermissions.Current.RequestPermissionsAsync(Permission.Storage);
 
             //Taking photo and storing it in MediaFile variable 'takenPhoto'
-            takenPhoto = await TakingPhotoHandler.TakePhoto();
+            try
+            {
+                takenPhoto = await TakingPhotoHandler.TakePhoto();
+            }
+
+            catch (ManagerException photoNotTakenException)
+            {
+                ToastUtil.ShowToast(photoNotTakenException.ErrorCode);
```

Comment:
You should probably allow retrying here?

Reply:
I allow it, after displaying this toast user can take photo again.

I know that there's not a lot to comment on, mainly XAML changes. Though thanks for the review, always appreciated!